### ISSUE TITLE
extend skin_logo to allow different images for differnt skins

### ISSUE THREAD

**Issue by johndoh** [roundcube/roundcubemail#6272]
Different skins can require different logos (colours/sizes/backgrounds) please consider extending the `skin_logo` config option to allow the use of different logos in different skins.

**Comment by alecpl** [comment]
Yup. How about adding skin prefix to the skin_logo array keys? E.g.:
```
array(
    "elastic:*" => "/images/logo.png",
    "larry:messageprint" => "/images/logo_print.png"
)
```
Would you create a PR?

**Comment by johndoh** [comment]
Sure. Do you think the order should be: 

- \<skin>:\<template>
- \<template>
- \<skin>:*
- \*

or

- \<skin>:\<template>
- \<skin>:*
- \<template>
- \*

**Comment by alecpl** [comment]
Good question. I think the later will be better.
